hack/lib/vars.bash

```diff
@@ -48,7 +48,10 @@ readonly UPGRADE_CLUSTER="${UPGRADE_CLUSTER:-"false"}"
 readonly UPGRADE_OCP_IMAGE="${UPGRADE_OCP_IMAGE:-}"
 
 readonly INSTALL_PREVIOUS_VERSION="${INSTALL_PREVIOUS_VERSION:-"false"}"
-export OLM_CHANNEL="${OLM_CHANNEL:-"candidate"}"
+export OLM_CHANNEL="${OLM_CHANNEL:-"4.5"}"
 # Change this when upgrades need switching to a different channel
 export OLM_UPGRADE_CHANNEL="${OLM_UPGRADE_CHANNEL:-"$OLM_CHANNEL"}"
 export OLM_SOURCE="${OLM_SOURCE:-"$OPERATOR"}"
+
+export CURRENT_CSV="serverless-operator.v1.8.0"
+export PREVIOUS_CSV="serverless-operator.v1.7.2"
```

Comment:
It'd be nice if you could derive the previous value from the CSV, using either `replaces` or `skipRange`. And if you can do that, grab the current one from there, too.